lib/connection.c

```diff
@@ -420,13 +425,30 @@ sf_bool STDCALL curl_post_call(SF_CONNECT *sf,
                                     SF_SQLSTATE_UNABLE_TO_CONNECT);
                 break;
             }
+
+            if(counter_for_code == 1 && sf->log_spec_info){
```

Comment:
shouldn't `counter_for_code == 1` be `counter_for_code == 1000`?

Reply:
Let's do it as following so that we don't keep incrementing `counter_for_code` indefinitely:
```
if (sf->log_spec_info){
  if(counter_for_code == 1000) {
    .... // logging
    }
  counter_for_code++;
}
```

Reply:
I changed to 1 to let me able to test it locally, will revert it back after some local tests